kaldb/src/main/java/com/slack/kaldb/chunk/ReadWriteChunkImpl.java

```diff
@@ -102,8 +102,14 @@ public void close() throws IOException {
     logSearcher.close();
     logStore.close();
     LOG.info("Closed chunk {}", chunkInfo);
-    logStore.cleanup();
-    LOG.info("Cleaned up chunk {}", chunkInfo);
+
+    try {
+      logStore.cleanup();
+      LOG.info("Cleaned up chunk {}", chunkInfo);
+    } catch (Exception e) {
+      // this will allow the service to still close successfully when failing to cleanup the file
+      LOG.error("Failed to cleanup logstore for chunk {}", chunkInfo);
```

Comment:
Should we also log the exception or is it logged already?

Reply:
Yep, fixed. 👍